### ISSUE TITLE
To do list functionality

### ISSUE THREAD

**Issue by felix-caboff** [GhostManager/Ghostwriter#395]
**Is your feature request related to a problem? Please describe.**
In an effort to work collaboratively, it would be really useful to have a place where you can dump things you want to follow up on.  We do use the notes functionality and this kinda works, but, can get a bit sprawling and chaotic.

**Describe the solution you'd like**
I'm thinking something similar to a To Do list where you can check things off.  This is maybe a big ask, but it would be really neat if this happened to be something that can be easily integrated.

**Describe alternatives you've considered**
We could just use a spreadsheet, but, the idea would be to work collaboratively.

**Comment by chrismaddalena** [comment]
Thanks for the suggestion! I'm open to the idea. We've kicked around similar ideas like project task checklists, recurring to do items, and features that act more like a scratch pad for collaborative notes/to do tracking. I'd like to do it.

**Comment by felix-caboff** [comment]
I think the last version of what you said sounds more appealing as it is a bit more free-form and can be used in many ways.  We want it so that we have a centralised place of things we need to revisit or follow up on.

**Comment by felix-caboff** [comment]
For future readers:

We have achieved this by adding an "extra field" at the report level that is just free-form text called "Task List".  It isn't "pretty" as such, but it functions just fine.  As this field isn't actively included in the report it doesn't go anywhere, but does act as that centralised space for us to dump things we know we need to revisit.

Closing as this is effectively resolved even if not explicitely.
